interface-definitions/firewall.xml.in

```diff
@@ -115,6 +115,79 @@
               #include <include/generic-description.xml.i>
             </children>
           </tagNode>
+          <node name="dynamic-group">
+            <properties>
+              <help>Firewall dynamic group</help>
+            </properties>
+            <children>
+              <tagNode name="address-group">
+                <properties>
+                  <help>Firewall dynamic address-group</help>
+                  <constraint>
+                    <regex>[a-zA-Z0-9][\w\-\.]*</regex>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="address">
+                    <properties>
+                      <help>Address-group member</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>IPv4 address to match. Range is not supported.</description>
```

Comment:
I don't think we should include "Range is not supported". `<format>ipv4</format> doesn't (or shouldn't ;) suggest that it is, and details belong to the docs (or to extended inline help, if/when we add such a capability)